VeganVirus/VeganVirus/ClipBoardAction.cpp

```diff
@@ -0,0 +1,60 @@
+#include "ClipBoardAction.h"
+
+
+ClipBoardAction::ClipBoardAction(double req) : Action(req, CLIP_BOARD_ICON_PATH)
+{
+}
+
+void ClipBoardAction::start()
+{
+	this->activeFlag = true;
+}
+
+void ClipBoardAction::update(double dt)
+{
+	if (!this->activeFlag)
+	{
+		return;
+	}
+	std::string data = this->ClipBoardRead();
+	if (data.find("meat") != std::string::npos)
+	{
+		this->clipBoardWrite("COWS ARE SMARTER THAN DOGS TEL_AVIV");
+	}
+	Sleep(100);
+}
+
```

Comment:
Spaces  are not consistent, choose a suitable profit and stick to it.

Reply:
which spaces? within the function? between the function?